src/components/@shared/ButtonBuy/index.tsx

```diff
@@ -57,6 +60,38 @@ function getConsumeHelpText(
   return text
 }
 
+function getAlgoHelpText(
```

Comment:
is the method used ?, could not find the call for it, if used sorry 😄

Reply:
yeah, probably something got missed up when merging 😅